securitas/controller/user.py

```diff
@@ -68,5 +69,6 @@ def user_edit(ipa, username):
     # Append 2 empty entries at the bottom of the fieldlist
     for i in range(2):
```

Comment:
Is this correct? If its intentional it might be confusing to have two empty fields originally. It also means if you actually have two keys you get 4 boxes with two being empty.
Would a better default be only show one empty box for each key and when one is added its number of keys + one empty field?

Reply:
yeah, for gpgkeys, we decided on 2 empty boxes. Probably for the SSH keys, one will work